sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -0,0 +1,334 @@
+        Returns a list of the sequence numbers of the enqueued messages.
+        :param message: The messages to schedule.
+        :type message: ~azure.servicebus.Message or ~azure.servicebus.BatchMessage
+        :param schedule_time_utc: The utc date and time to enqueue the messages.
+        :type schedule_time_utc: ~datetime.datetime
+        :rtype: List[int]
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sync_samples/sample_code_servicebus.py
+                :start-after: [START scheduling_messages]
+                :end-before: [END scheduling_messages]
+                :language: python
+                :dedent: 4
+                :caption: Schedule a message to be sent in future
+        """
+        # pylint: disable=protected-access
+        self._open()
+        if isinstance(message, BatchMessage):
+            request_body = self._build_schedule_request(schedule_time_utc, *message._messages)
+        else:
+            request_body = self._build_schedule_request(schedule_time_utc, message)
+        return self._mgmt_request_response_with_retry(
+            REQUEST_RESPONSE_SCHEDULE_MESSAGE_OPERATION,
+            request_body,
+            mgmt_handlers.schedule_op
+        )
+
+    def _cancel_scheduled_messages(self, sequence_numbers):
+        # type: (Union[int, List[int]]) -> None
+        """
+        Cancel one or more messages that have previously been scheduled and are still pending.
+
+        :param sequence_numbers: The sequence numbers of the scheduled messages.
+        :type sequence_numbers: int or list[int]
+        :rtype: None
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sync_samples/sample_code_servicebus.py
+                :start-after: [START cancel_scheduled_messages]
+                :end-before: [END cancel_scheduled_messages]
+                :language: python
+                :dedent: 4
+                :caption: Cancelling messages scheduled to be sent in future
+        """
+        self._open()
+        if isinstance(sequence_numbers, int):
+            numbers = [types.AMQPLong(sequence_numbers)]
+        else:
+            numbers = [types.AMQPLong(s) for s in sequence_numbers]
+        request_body = {'sequence-numbers': types.AMQPArray(numbers)}
+        return self._mgmt_request_response_with_retry(
+            REQUEST_RESPONSE_CANCEL_SCHEDULED_MESSAGE_OPERATION,
+            request_body,
+            mgmt_handlers.default
+        )
+
+    @classmethod
+    def from_connection_string(
+        cls,
+        conn_str,
+        **kwargs
+    ):
+        # type: (str, Any) -> ServiceBusSender
+        """Create a ServiceBusSender from a connection string.
+
+        :param conn_str: The connection string of a Service Bus.
+        :keyword str queue_name: The path of specific Service Bus Queue the client connects to.
+        :keyword str topic_name: The path of specific Service Bus Topic the client connects to.
```

Comment:
Would probably make a note here (and similarly for receiver if it uses this same wording) at the mutual-exclusivity of these options.